README.md

```diff
@@ -1 +1,46 @@
-# medical-database
\ No newline at end of file
+# medical-database
+in this project a database is implemented from the following diagram for a clinic:
+
+![image](https://user-images.githubusercontent.com/78906545/167815262-de546506-6d3a-482d-bf6f-31e5c3ee87c4.png)
+## Getting Started
+
+This repository includes files with plain SQL that can be used to recreate a database:
+
+- Use [schema.sql](./schema.sql) to create all tables.
+**Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+
+## Author 1
+
+👤 **Okoroji Victor aka Vickymarz**
+
+- GitHub: [@githubhandle](https://github.com/vickymarz)
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/okoroji-victor-ebube-8791741a0)
+
+## Author 2
+
+👤 **Zahra Arshia**
+
+- GitHub: [@githubhandle](https://github.com/ZahraArshia)
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zahra-arshia/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## Acknowledgments
+
+- Microverse 
+- Coding partner
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The MIT link is broken. I would recommend fixing it to make README professional.

Reply:
Nice observation. Thanks